sources/b12/sections/00-introduction.adoc

```diff
@@ -27,22 +27,18 @@ informative, as well, although they are not the primary audience.
 
 === Document Structure
 
-This document addresses several topics related to crowdsourced bathymetry. Chapter One, “Data
-Contribution,” provides information about how to send crowdsourced bathymetry to the IHO Data Centre
+This document addresses several topics related to crowdsourced bathymetry. Chapter One, “<<data-contribution>>,” provides information about how to send crowdsourced bathymetry to the IHO Data Centre
```

Comment:
I think `Chapter One "<<data-contribution>>"` will be rendered as `Chapter One "Clause X"` rather than `Chapter One "Data Contribution"`. Can we do `<<data-contribution>>, "Data Contribution"` instead?

Reply:
Excuse me @ronaldtse, just to be sure, you mean to put: `<<data-contribution>>,"Data Contribution"`, or , `<<data-contribution,Data Contribution>>` ?

Reply:
Excuse me @ronaldtse, just to be sure, do you mean to put `<<data-contribution>>,"Data Contribution"` , or, `<<data-contribution, Data Contribution>>`?
(Wouldn't the second option be preferable?)

Reply:
First option taken with a little tweak, `"<<data-contribution>>, Data Contribution"`

Reply:
Typically yes, but here it includes both the location and the title. In Metanorma we do not have a mechanism for citing the "title", only the "location".

In Metanorma if you do `<<data-contribution>>`, it render the location. If you want "Chapter One 'Data Contribution'", it is preferable to write `<<data-contribution>>` for "Chapter One" so the location is cited, then the title manually.